scripts/gufi_stats

```diff
@@ -993,7 +993,7 @@ def uid_size(_config, args, where):
 def gid_size(_config, args, where):
     return uidgid_size(args, 'gid', where)
 
-def size_bins(args, base, type): # pylint: disable=redefined-builtin
+def size_bins(args, base, type, field='size'): # pylint: disable=redefined-builtin
```

Comment:
lets not have default arguments. the only other place there are default arguments is `build_where`, and they arent even used